Sources/Client/Model/EventType.swift

```diff
@@ -0,0 +1,115 @@
+    
+    /// When a channel was updated (when watching the channel).
+    case channelUpdated = "channel.updated"
+    /// When a channel was deleted (when watching the channel).
+    case channelDeleted = "channel.deleted"
+    /// When a channel was hidden (when watching the channel).
+    case channelHidden = "channel.hidden"
+    
+    /// When a new message was added on a channel (when watching the channel).
+    case messageNew = "message.new"
+    /// When a message was updated (when watching the channel).
+    case messageUpdated = "message.updated"
+    /// When a message was deleted (when watching the channel).
+    case messageDeleted = "message.deleted"
+    /// When a channel was marked as read (when watching the channel).
+    case messageRead = "message.read"
+    /// ⚠️ When a message reaction was added or deleted (when watching the channel).
+//    case messageReaction = "message.reaction"
+    
+    /// When a member was added to a channel (when watching the channel).
+    case memberAdded = "member.added"
+    /// When a member was updated (when watching the channel).
+    case memberUpdated = "member.updated"
+    /// When a member was removed from a channel (when watching the channel).
+    case memberRemoved = "member.removed"
+    
+    /// When a message reaction was added.
+    case reactionNew = "reaction.new"
+    /// When a message reaction updated.
+    case reactionUpdated = "reaction.updated"
+    /// When a message reaction deleted.
+    case reactionDeleted = "reaction.deleted"
+    
+    /// When a message was added to a channel (when clients that are not currently watching the channel).
+    case notificationMessageNew = "notification.message_new"
+    /// When the total count of unread messages (across all channels the user is a member) changes
+    /// (when clients from the user affected by the change).
+    case notificationMarkRead = "notification.mark_read"
+    /// When the user mutes someone.
+    case notificationMutesUpdated = "notification.mutes_updated"
+    
+    /// When the user accepts an invite (when the user invited).
+    case notificationAddedToChannel = "notification.added_to_channel"
+    /// When a user was removed from a channel (when the user invited).
+    case notificationRemovedFromChannel = "notification.removed_from_channel"
+    
+    /// When the user was invited to join a channel (when the user invited).
+    case notificationInvited = "notification.invited"
+    /// When the user accepts an invite (when the user invited).
+    case notificationInviteAccepted = "notification.invite_accepted"
+    /// When the user reject an invite (when the user invited).
+    case notificationInviteRejected = "notification.invite_rejected"
+    
+    /// Checks if the type is a channel event type.
+    public var isChannelCase: Bool {
+        EventType.channelCases.contains(self)
+    }
+    
+    /// All channel event types.
+    public static var channelCases: Set<EventType> {
```

Comment:
`channelEventTypes` would make more sense than `Cases`, same for `isChannelEventType`

Reply:
Fixed